README.md

```diff
@@ -7,3 +7,16 @@ This is a suitcase subpackage for writing a particular file format.
 ```
 pip install suitcase-msgpack
 ```
+
+## Quick Start
+
+```
+import suitcase.msgpack
+docs = db[-1].documents(fill=True)
+suitcase.msgpack.export(docs, 'my_exported_files/')
+```
+The exported file will have path `my_exported_files/8fe81a89-8103-487a-85d2-23c70af96f16-primary.msgpack`. The default file prefix can be changed with the `file_prefix` keyword argument. See documentation link below.
```

Comment:
This line is too long. Please make it 80 symbols wide (<kbd>Shift</kbd>+<kbd>V</kbd> and then <kbd>gq</kbd> in vim).
```suggestion
The exported file will have path `my_exported_files/8fe81a89-8103-487a-85d2-23c70af96f16-primary.msgpack`. The default file prefix can be changed with the `file_prefix` keyword argument. See the documentation link below.
```